### ISSUE TITLE
macOS release 2018.710 cannot connect to internet

### ISSUE THREAD

**Issue by KevZhi** [ppy/osu#3018]
Logging and osu!direct is both failed, online chat is also disabled.

osu!lazer macOS release 2018.710
macOS Mojave Developers Beta 3

network.log
```
07/12/2018 11:32:52: ----------------------------------------------------------
07/12/2018 11:32:52: Network Log for kevin (LogLevel: Verbose)
07/12/2018 11:32:52: osu 2018.710.0.0
07/12/2018 11:32:52: Running on Unix 18.0.0.0, 4 cores
07/12/2018 11:32:52: ----------------------------------------------------------
07/12/2018 11:32:52: Request to https://api.github.com/repos/ppy/osu/releases/latest successfully completed!
07/12/2018 11:33:30: We just went Connecting!
07/12/2018 11:33:31: Request to https://osu.ppy.sh/oauth/token failed with System.Net.WebException: Unauthorized.
07/12/2018 11:33:31: Login failed!
07/12/2018 11:33:31: We just went Offline!
```

Glad to provide any other information if needed.

![2018-07-12 19 28 18](https://user-images.githubusercontent.com/32162677/42630570-c5f6002e-8609-11e8-9ddf-6534e771728c.png)

**Comment by KevZhi** [comment]
Pretty sure, i tested email+password/username+password. Passwords are absolutely correct. It didn’t notify me if password is right or wrong. And other features needs connection also disabled.
I’m wondering if it’s stucked on two step email auth which lazer don’t support now

支天 | Kevin Zhi

________________________________
收件人: Dean Herbert <notifications@github.com>
发送时间: 星期五, 七月 13, 2018 18:25
收件人: ppy/osu
抄送: Kevin Zhi; Author
主题: Re: [ppy/osu] macOS release 2018.710 cannot connect to internet (#3018)


Are you sure your login is correct?

―

**Comment by peppy** [comment]
What happens in the game interface when it fails? Does it ask you to enter your login again or freeze on "connecting"?

**Comment by KevZhi** [comment]
@peppy 
![imb_7x49nl](https://user-images.githubusercontent.com/32162677/42690671-c071eba8-86d7-11e8-9ebf-a1bee8332f2c.GIF)
it didn't ask me to login again nor freeze on connecting. 
the password field was cleaned, just like back to before I clicked the sign in button

**Comment by tim241** [comment]
same issue on linux

**Comment by tim241** [comment]
huh, I tried again and it worked.

**Comment by dgel** [comment]
@jacksonisiah I was afraid of that. Well, I'll see if I can swing getting registered some other way. Thanks for the prompt reply!

**Comment by KevZhi** [comment]
Yep on 801version still cannot login, gonna provide you details later, is there any macOS users works fine?

支天 | Kevin Zhi

________________________________
收件人: Dean Herbert <notifications@github.com>
发送时间: 星期四, 八月 2, 2018 12:22
收件人: ppy/osu
抄送: Kevin Zhi; Mention
主题: Re: [ppy/osu] macOS release 2018.710 cannot connect to internet (#3018)


@KevZhi<https://github.com/KevZhi> is this still happening in the latest build?

―

**Comment by peppy** [comment]
It still works fine locally for me on both my PCs.

**Comment by KevZhi** [comment]
@peppy the error log and detail are totally same.

Request to https://osu.ppy.sh/oauth/token failed with System.Net.WebException: Unauthorized.

I'm starting thinking about if the login is failed because it need a email verification but lazer doesn't support the system.

As a Chinese player you know, we always need to fly across earth 10rounds a day to connect to a good proxy, our ip address locates changes will be faster than plane so no wonder I triggered the email auth.

**Comment by KevZhi** [comment]
Yesterday I did a clean reinstall on macOS mojave, problem still exist ,so I can almost make sure it is because email auth system

**Comment by KevZhi** [comment]
fixed on 11.08 ver.
may because of email auth triggered but lazer doesn't support email auth now.
